app/views/posts/index.html.erb

```diff
@@ -1 +1,39 @@
-<h1> display list of posts </h1>
\ No newline at end of file
+<section class="users-container">
+  <% user = User.where(id: params[:user_id])[0] %>
```

Comment:
- [ ] I'd suggest moving the logic responsible for getting the user data to its corresponding controller action, controller logic should be separated from rendering views. Should be something like this:
```ruby
@users = User.all
```
then you can access `@users` directly from the view and render its data.

Reply:
thanks for your awesome feedback :heart_decoration: